config/initializers/menus.rb

```diff
@@ -48,6 +55,9 @@
 end
 
 Redmine::MenuManager.map :account_menu do |menu|
+  menu.push :administration,
+            { controller: '/admin', action: 'projects' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.